move.py

```diff
@@ -27,3 +29,17 @@ def draft(cls, moves):
         cls.write(moves, {
             'state': 'draft',
             })
+
+    @classmethod
+    def delete(cls, moves):
+        invoices = [move for move in moves if move.origin
+            and move.origin.__name__ == 'account.invoice'
+            and move.origin.state != 'draft']
```

Comment:
In case an invoice is draft, is not related with a account.move. Maybe is not necessary to check origin.state, and raise when delete a move and this is related with an invoice.